_javascript/curriculum.js

```diff
@@ -94,6 +94,9 @@ $(function(){
 
         updateSlideSize();
       }
+     //add error from ajax call
+      error: function (xhr) {
```

Comment:
What is the error method used for if it does not have a body?
Not having it fulfills the same purpose.

Reply:
@JasonEtco Do you have any ideas here?

Reply:
What @stoe said -  not having the method at all is functionally the same as having an empty one.